comment/locale/hi/LC_MESSAGES/djangojs.po

```diff
@@ -17,58 +17,63 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-#: comment/static/js/comment.js:145 comment/static/js/comment.js:286
+
+#: comment/static/js/comment.js:151 comment/static/js/comment.js:291
 msgid "Replies"
 msgstr "जवाब"
 
-#: comment/static/js/comment.js:147 comment/static/js/comment.js:288
+#: comment/static/js/comment.js:153 comment/static/js/comment.js:293
 msgid "Reply"
 msgstr "जवाब दें"
 
-#: comment/static/js/comment.js:164
+#: comment/static/js/comment.js:170
 msgid "Unable to post your comment!, please try again"
 msgstr "आपकी टिप्पणी पोस्ट करने में असमर्थ!, कृपया पुनः प्रयास करें"
 
-#: comment/static/js/comment.js:185
+#: comment/static/js/comment.js:191
 msgid "You can't edit this comment"
 msgstr "आप इस टिप्पणी को बदल नहीं सकते"
 
-#: comment/static/js/comment.js:208
+#: comment/static/js/comment.js:214
 msgid "Modification didn't take effect!, please try again"
 msgstr "संशोधन प्रभावी नहीं हुआ!, कृपया पुनः प्रयास करें"
 
-#: comment/static/js/comment.js:298
+#: comment/static/js/comment.js:302
 msgid "Unable to delete your comment!, please try again"
 msgstr "आपकी टिप्पणी को हटाने में असमर्थ!, कृपया पुनः प्रयास करें"
 
-#: comment/static/js/comment.js:355
+#: comment/static/js/comment.js:359
 msgid "Reaction couldn't be processed!, please try again"
 msgstr "प्रतिक्रिया संसाधित नहीं की जा सकी!, कृपया पुनः प्रयास करें"
 
-#: comment/static/js/comment.js:462
+#: comment/static/js/comment.js:394
+msgid "Subscription couldn't be processed!, please try again"
+msgstr "सदस्यता संसाधित नहीं हो सकी!, कृपया पुनः प्रयास करें"
+
```

Comment:
Have you added the translation for these? or is `django` trying to be smart?

Reply:
I added with google translation 😄 This can be adjusted later in case of bad translation. But this is needed in the development now to make sure that every string tagged for translation is translated.

Reply:
Oh !!! I don't think this is a great decision especially for languages that both of us might not be aware of. This may cause an issue to newcomers too.

Reply:
In fact using a language that has no ascii letter at all is a good choice to make sure that the string is translated.
This is the method that checks for translation:
https://github.com/Radi85/Comment/pull/127/files/e81dd3dd749bb16afccea6f4d139ac039c6622cb#diff-c2eb376f4cca55dbd53c16353ae9670ae28e0a9e50d73b2f5676ad105726fb30R40-R45

Reply:
no, my point is that because there was only 1 language at present, you could do this manually and on your own. What if the applications grows to more audience?

Reply:
So far we are using locale `hi` as a part of our testing procedure to make sure the all strings are added to .po file and are translated. 
This will not test if the new added string is alos added to .po file of other language. For that we can think about different way to test that every string added to app is also added to .po of other languages.
Since the .po file in `hi` is tested we can make it as a start point for us to compare other files with it.

Reply:
there are a couple of issues that come to my mind if we go on to that route.

- internationalization is part of the development and not translation. this just makes translation too as a part of it.
- this makes the process cumbersome. The author has to add translations for all available languages when a change in string is made to the code. 
- The author obviously has a knowledge of very limited number of languages and google translation or any other service is not really sensitive to the intricacies of a language. In this situation I would prefer to have the translation at an earlier version.

Reply:
Cool.
I will generate .po for locale `test` and activate it in the test and we use it to test the translation, we just make sure that we use none ascii letters in it.
In this case we won't interfer with the language translation but this will make sure the tring is translated.
I have already started this implementation, you can adjust hi language translation after merging this PR and there won't be any issue with test.

Reply:
I will revert the .po changes in hi language to avoid the issue mentioned above. we are using no test langauge instead.

Reply:
no, the changes seem fine to me at a glance. your hindi seems to be great :stuck_out_tongue_closed_eyes: